Foundation/ExtraStringAPIs.swift

```diff
@@ -13,19 +13,9 @@
 // Random access for String.UTF16View, only when Foundation is
 // imported.  Making this API dependent on Foundation decouples the
 // Swift core from a UTF16 representation.
-extension String.UTF16View.Index : Strideable {
```

Comment:
This is potentially a source break, right? @moiseev do we have any way of deprecating conformance?